src/main/kotlin/uk/gov/justice/digital/hmpps/bodyscan/apiclient/model/PersonalCareNeeds.kt

```diff
@@ -0,0 +1,37 @@
+package uk.gov.justice.digital.hmpps.bodyscan.apiclient.model
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonProperty
+import uk.gov.justice.digital.hmpps.bodyscan.model.BodyScanReason
+import uk.gov.justice.digital.hmpps.bodyscan.model.BodyScanResult
+import java.time.LocalDate
+
+data class PersonalCareNeeds(
+  @JsonIgnore
+  val date: LocalDate,
+  @JsonIgnore
+  val bodyScanReason: BodyScanReason,
+  @JsonIgnore
+  val bodyScanResult: BodyScanResult
+
+) {
+  @JsonProperty("problemCode")
+  fun getProblemCode() = "BSC6.0"
+
+  @JsonProperty("problemStatus")
+  fun getProblemStatus() = "ON"
+
+  @JsonProperty("startDate")
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+  fun getStartDate() = date
+
+  @JsonProperty("endDate")
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
+  fun getEndDate() = date
+
+  @JsonProperty("commentText")
+  fun getCommentText() = "${bodyScanReason.toString().lowercase().replaceFirstChar { it.uppercase() }}" +
+    "-" +
+    "${bodyScanResult.toString().lowercase().replaceFirstChar { it.uppercase() }}"
```

Comment:
don't think we needs to upper case the first character of the result bit so can just  `.replaceFirstChar { it.uppercase() }}` on the entire string

On DCS-1727 theres an example

Reply:
sorted